opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -326,9 +326,9 @@ namespace Opm {
 
         well_state_ = previous_well_state_;
         well_state_.disableGliftOptimization();
+        updateAndCommunicateGroupData();
```

Comment:
Could the movement of this function to here, explain the failing tests?

Reply:
yes. but I think moving it here is correct. The old code used old group data in the welltest and well potential code etc.